frontend/packages/yki/src/components/skeletons/PublicEvaluationPeriodListingSkeleton.tsx

```diff
@@ -0,0 +1,47 @@
+import { TableCell, TableRow } from '@mui/material';
+import { CustomSkeleton, H3, NormalTable } from 'shared/components';
+import { SkeletonVariant } from 'shared/enums';
+
+import { PublicEvaluationPeriodListingHeader } from 'components/reassessment/PublicEvaluationPeriodListingHeader';
+import { useCommonTranslation, usePublicTranslation } from 'configs/i18n';
+
+const getRowDetails = () => (
+  <TableRow>
+    <TableCell></TableCell>
+    <TableCell></TableCell>
+    <TableCell></TableCell>
+    <TableCell></TableCell>
+  </TableRow>
+);
+
+export const PublicEvaluationPeriodListingSkeleton = () => {
+  const translateCommon = useCommonTranslation();
+  const { t } = usePublicTranslation({
+    keyPrefix: 'yki.component.publicEvaluationPeriodListing',
+  });
+
+  return (
+    <>
+      <CustomSkeleton
```

Comment:
Selaimella itse en saa näkyviin muuta kuin sinisen taustan, joka sitten vaihtuu tutkintojen listaukseksi kun nuo listattavat tutkinnot saadaan backendiltä tai proxyltä palautettua. Näin se varmaan pitäiskin mennä, mietin vaan pystyykö täälläkään määriteltyä sisältöä oikeasti näkemään, esim. tuo heading. Tai näkyvätkö esim. ruudunlukijalla.

Reply:
Joo, ei noiden oikein pitäisikään muuta kuin vain jotain taustaväriä rendata. Skeletoneille annetun sisällön pitäisi lähinnä ohjata rendattavan skeletonin kokoa. Ruudunlukijalle kuvittelisi tuon ariaLabelin tulevan tietoon.